rust/src/delta.rs

```diff
@@ -207,6 +207,29 @@ pub struct DeltaTableMetaData {
     pub configuration: HashMap<String, String>,
 }
 
+impl DeltaTableMetaData {
+    fn new(
+        name: Option<String>, 
+        description: Option<String>, 
+        schema: Schema, 
+        partition: Vec<String>, 
+        config: HashMap<String, String>
+    ) -> Self {
+        //use inputs
+        
+        Self {
+            id: Uuid::new_v4().to_string(), //create a new GUID
+            name: name,
+            description: description,
+            format: action::Format::new("parquet".to_string(), None),
+            schema: schema,
+            partition_columns: partition,
+            created_time: Utc::now().timestamp(),  //create a timestamp for current timestamp
```

Comment:
The timestamp is in millisecond precision: https://github.com/delta-io/delta/blob/master/PROTOCOL.md#change-metadata, https://github.com/delta-io/delta/blob/master/core/src/main/scala/org/apache/spark/sql/delta/actions/actions.scala#L358.

You could use this method: https://docs.rs/chrono/0.4.19/chrono/struct.DateTime.html#method.timestamp_millis